x-pack/plugins/apm/public/components/app/mobile/service_overview/geo_map/map_layers/get_layer_list.ts

```diff
@@ -13,16 +13,18 @@ import { MapTypes } from '../../../../../../../common/mobile/constants';
 export async function getLayerList({
   selectedMap,
   maps,
+  dataViewId,
 }: {
   selectedMap: MapTypes;
-  maps?: MapsStartApi;
+  maps: MapsStartApi | undefined;
+  dataViewId: string;
 }): Promise<LayerDescriptor[]> {
   switch (selectedMap) {
     case MapTypes.Http:
-      return await getHttpRequestsLayerList(maps);
+      return await getHttpRequestsLayerList(maps, dataViewId);
```

Comment:
I think you can just remove the `await` here, and leave it to the caller of this function to await the response.